src/test/java/org/mybatis/dynamic/sql/update/UpdateStatementTest.java

```diff
@@ -171,6 +171,29 @@ public void testFullUpdateStatementNoWhere() {
         });
     }
     
+    @Test
+    public void testUpdateStatementDynamicArithmeticOperation() {
+        UpdateStatementProvider updateStatement = update(foo)
+                .set(id).incrementBy()
+                .set(id).decrementBy()
+                .set(id).multiplyBy()
+                .set(id).divideBy()
+                .build()
+                .render(RenderingStrategy.MYBATIS3);
+        
+        String expectedStatement = "update foo " 
+                + "set id = id + #{parameters.p1,jdbcType=INTEGER}, "
+                + "id = id - #{parameters.p2,jdbcType=INTEGER}, "
+                + "id = id * #{parameters.p3,jdbcType=INTEGER}, "
+                + "id = id / #{parameters.p4,jdbcType=INTEGER}";
+                
+        SoftAssertions.assertSoftly(softly -> {
+            softly.assertThat(updateStatement.getUpdateStatement()).isEqualTo(expectedStatement);
+        
+            softly.assertThat(updateStatement.getParameters().size()).isEqualTo(0);
```

Comment:
This is not right.  If there are 4 parameter markers, there should be 4 parameters.